conda/plugins/hookspec.py

```diff
@@ -332,3 +333,48 @@ def conda_settings():
                    aliases=("example_option_alias",),
                )
         """
+
+    @_hookspec
+    def conda_transaction_action(self) -> Iterable[CondaTransactionAction]:
```

Comment:
Per pluggy parlance hooks should be plural since they are implemented not as singular instances but always as a list, or in our case generators as we're using `yield`